diff -up grommunio-admin-api-1.18/CMakeLists.txt.omv~ grommunio-admin-api-1.18/CMakeLists.txt
--- grommunio-admin-api-1.18/CMakeLists.txt.omv~	2026-03-13 17:17:46.403800327 +0100
+++ grommunio-admin-api-1.18/CMakeLists.txt	2026-03-13 17:17:55.093171670 +0100
@@ -17,7 +17,7 @@ include(GNUInstallDirs)
 
 find_package(PkgConfig REQUIRED)
 
-set(PHP_FPM_SERVICE "php-fpm.service" CACHE STRING "Name of the PHP FPM service")
+set(PHP_FPM_SERVICE "php-fpm@grommunio.service" CACHE STRING "Name of the PHP FPM service")
 
 set(COMMONCONFDIR ${CMAKE_INSTALL_FULL_SYSCONFDIR}/grommunio-admin-common)
 set(COMMONDIR ${CMAKE_INSTALL_FULL_DATADIR}/grommunio-admin-common)
diff -up grommunio-admin-api-1.18/data/config.yaml.in.omv~ grommunio-admin-api-1.18/data/config.yaml.in
--- grommunio-admin-api-1.18/data/config.yaml.in.omv~	2026-03-13 19:26:02.130440175 +0100
+++ grommunio-admin-api-1.18/data/config.yaml.in	2026-03-13 19:26:31.198599638 +0100
@@ -14,7 +14,7 @@ options:
     local: 'http://localhost:8080/statusmonitor'
   dashboard:
     services:
-      - unit: grommunio-antispam.service
+      - unit: rspamd.service
       - unit: gromox-delivery.service
       - unit: gromox-event.service
       - unit: gromox-http.service
@@ -37,7 +37,7 @@ logs:
   Admin API:
     source: 'grommunio-admin-api.service'
   Antispam:
-    source: 'grommunio-antispam.service'
+    source: 'rspamd.service'
   Gromox http:
     source: 'gromox-http.service'
   Gromox smtp:
diff -up grommunio-admin-api-1.18/data/polkit-10-grommunio.rules.omv~ grommunio-admin-api-1.18/data/polkit-10-grommunio.rules
--- grommunio-admin-api-1.18/data/polkit-10-grommunio.rules.omv~	2026-03-13 17:18:40.805681359 +0100
+++ grommunio-admin-api-1.18/data/polkit-10-grommunio.rules	2026-03-13 19:25:27.489686033 +0100
@@ -2,7 +2,7 @@ polkit.addRule(function(action, subject)
 	const services = [
 		"grommunio-admin-api.service",
 		"grommunio-admin-api.socket",
-		"grommunio-antispam.service",
+		"rspamd.service",
 		"grommunio-cui@tty1.service",
 		"grommunio-fetchmail.timer",
 		"grommunio-index.timer",
@@ -16,7 +16,7 @@ polkit.addRule(function(action, subject)
 		"gromox-timer.service",
 		"gromox-zcore.service",
 		"nginx.service",
-		"php-fpm.service",
+		"php-fpm@grommunio.service",
 		"postfix.service",
 		"redis@grommunio.service",
 	];
